sql/telemetry/clients_daily_histogram_aggregates_v1/view.sql

```diff
@@ -0,0 +1 @@
+SELECT * FROM telemetry_derived.clients_daily_histogram_aggregates_v1
```

Comment:
It's a BQ-level restriction that views must always use fully-qualified names. And we also require that `view.sql` files explicitly contain `CREATE OR REPLACE VIEW` statements, though I would like to eventually move to this style where the file contains only the body of the query.

```suggestion
CREATE OR REPLACE VIEW
  `moz-fx-data-shared-prod.telemetry.clients_daily_histogram_aggregates`
AS SELECT 
  * 
FROM 
  `moz-fx-data-shared-prod.telemetry_derived.clients_daily_histogram_aggregates`
```

Reply:
The automated schema pipeline handles publishing views and will also publish the views to the derived-datasets project so that Redash can see it.

Reply:
I've updated these, but the `dry-run-sql` script seems to be failing. I believe it's because in the views it's looking for the tables in `moz-fx-data-shared-prod` which don't exist and unfortunately I don't have permission to create them.

Would it make more sense to reference `moz-fx-data-derived-datasets` for now instead?

Reply:
Yes, @fbertsch has reviewed the tables in earlier PRs. I had filed an issue here to create the tables from the contents of this PR: https://github.com/mozilla/bigquery-etl/pull/455. Basically only 2 of the tables have partitions and clustering that needs to be specified (for now anyways) so they needed to be explicitly created with that.

The other tables I figured could be created in the airflow dag: https://github.com/mozilla/telemetry-airflow/pull/607 - note this needs a bit of an update after this PR..

Either way, I think it doesn't hurt to make all the tables in advance.

Reply:
I will try to make this happen right now.